gremlin-test/features/filter/Dedup.feature

```diff
@@ -306,4 +306,15 @@ Feature: Step - dedup()
       | ripple |
       | peter |
       | marko |
-      | josh  |
\ No newline at end of file
+      | josh  |
+
+    Scenario: g_injectX1X_dedupXlocalX_unfold
```

Comment:
this was the contrived example from the JIRA, but i think it comes with the downside that you had to add OptOut for them in computer tests. could you come up with tests that don't require that OptOut please?